api/src/main/java/com/lovely4k/backend/diary/service/DiaryService.java

```diff
@@ -73,4 +73,11 @@ private Diary validateDiaryId(Long diaryId) {
         );
     }
 
+    @Transactional
+    public void deleteDiary(Long diaryId, Long memberId) {
+        Diary diary = validateDiaryId(diaryId);
+        Member member = validateMemberId(memberId);
+        diary.checkAuthority(member.getCoupleId());
```

Comment:
```suggestion
        Diary diary = findDiary(diaryId, memberId);
```
쿼리로 한번에 해결은 어려울까요? 2번의 쿼리와 checkAuthority로 별도 검사하는건 응집도가 떨어지는거같아요

Reply:
감사합니다. 해당 부분 수정해보도록 하겠습니다!!